tools/TZupdate.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+dir=$(cd ${0%/*}; pwd)
```

Comment:
it should be noted on how to run this script? if ran from tools/, this will show `TZupdate.sh: line 3: cd: TZupdate.sh: Not a directory`
or $1 should be output?

Reply:
It is supposed to work from anywhere (`./`, `tools/` or `/path/to/`) without arguments.
What is your shell ? (I tested with bash and dash)

Reply:
The trick is to run it as `sh TZupdate.h`. I have tried with Ubuntu install, both dash and bash. `set -x` shows that the `./` will do `cd .` and `sh` call will try to cd into the script name

Reply:
Thanks for pointing me to this use case. It is harmless, it is now hidden.
`cd` works when the script is called as `./script` or `path/to/script` and also when called with `sh tools/script`. When called with `sh script`, `cd` throws an error (now hidden) but `pwd` returns the right subdir.